build.sbt

```diff
@@ -59,7 +59,8 @@ lazy val client: Project = (project in file("client"))
     resolvers += Resolver.defaultLocal,
     // use uTest framework for tests
     testFrameworks += new TestFramework("utest.runner.Framework"),
-    scalaJSUseMainModuleInitializer := true
+    scalaJSUseMainModuleInitializer := true,
+    parallelExecution in Test := false
```

Comment:
Does this make running the tests slower?

Reply:
8 seconds slower, but it means we have readable test logs